mozci/task.py

```diff
@@ -262,33 +262,54 @@ def _load_errorsummary(self):
             return
 
         groups = None
+        group_results = {}
 
         lines = [json.loads(l) for l in self.get_artifact(path).splitlines()]
+
+        has_group_result = any(line["action"] == "group_result" for line in lines)
+
         for line in lines:
             if line["action"] == "test_groups":
                 groups = list(set(line["groups"]) - {"default"})
 
-            elif line["action"] == "test_result":
+            # After enough time has passed since https://bugzilla.mozilla.org/show_bug.cgi?id=1632242,
+            # we can switch to look for `group_result` lines exclusively.
```

Comment:
Rather than linking to the bug, just add a `TODO` with a date, e.g:
```
# TODO After August 1st 2020, switch to using group_result exclusively.
```